src/bot.ts

```diff
@@ -186,11 +185,11 @@ export class DiscordBot {
         });
 
         // Due to messages often arriving before we get a response from the send call,
-        // messages get delayed from discord. We use Bluebird.delay to handle this.
+        // messages get delayed from discord. We use Util.Delay to handle this.
```

Comment:
what is it now? `Util.Delay` or `Util.DelayedPromise`? be consistent ^-^

Reply:
oopise, yes